README.md

```diff
@@ -7,10 +7,10 @@ Build a simple dockerised web app that meets the requirements listed below.
 - Use any language you like
 - Should respond to `/`, `/treasure` and `/_health`
 	- `/`: Prints 'Hello World' in any design you like. Returns 200
-	- `/treasure`: Show what treasure exists (based on configuration) in any design you like. Returns 200
-	- `/_health`: Returns a JSON-formatted message saying whether everything is OK. Returns 200
+	- `/treasure`: Display a configurable string as a UI or structured API response. Returns 200
```

Comment:
Would we use "in" or "as part of" rather than "as"? I think that would make it a little clearer that the configured value should be _included in_ the response, rather than constituting the entire response.

Reply:
Is "configurable string" ambiguous?